modules/cluster-logging-collector-external.adoc

```diff
@@ -6,7 +6,7 @@
 = Configuring Fluentd to send logs to an external log aggregator
 
 You can configure Fluentd to send a copy of its logs to an external log
-aggregator, and not the default Elasticsearch, using the `forward`
+aggregator, and not the default Elasticsearch, using the *out-forward*
```

Comment:
*out_forward* - underscore, not dash - https://docs.fluentd.org/output/forward "The out_forward Buffered Output plugin forwards events to other fluentd nodes"